includes/class-wcs-object-data-cache-manager.php

```diff
@@ -196,6 +206,84 @@ public function action_object_cache_changes( $object ) {
 		}
 	}
 
+	/**
+	 * When an object is restored from the trash, action on object changes.
+	 *
+	 * @param int $object_id The object id being restored.
+	 */
+	public function untrashed( $object_id ) {
+		$object = $this->get_object( $object_id );
+		if ( null === $object ) {
+			return;
+		}
+
+		$this->action_object_cache_changes( $object );
+	}
+
+	/**
+	 * When an object is to be deleted, prepare object changes to update all fields
+	 * and mark those changes as deletes.
+	 *
+	 * @param int   $object_id The id of the object being deleted.
+	 * @param mixed $object    The object being deleted.
+	 */
+	public function prepare_object_to_be_deleted( $object_id, $object ) {
+		if ( ! $object->get_id() ) {
+			return;
+		}
+
+		$this->prepare_object_changes( $object, 'all_fields', true );
```

Comment:
```suggestion
		$this->prepare_object_changes( $object, 'all_fields' );
```

No third parameter anymore.